language-server/modules/langserver-core/src/test/resources/references/project/pkg1/tests/test1.bal

```diff
@@ -0,0 +1,33 @@
+import ballerina/io;
```

Comment:
Shall we add the license header?

Reply:
This is only a test source since we neglect the license header